inc/api/class-wprus-api-abstract.php

```diff
@@ -349,7 +355,13 @@ public function authorize_notification() {
 			} elseif ( 'post' === $this->method ) {
 
 				if ( self::$ip_whitelist ) {
-					$is_authorized_remote = in_array( $_SERVER['REMOTE_ADDR'], self::$ip_whitelist, true );
+					foreach ( self::$ip_whitelist as $range ) {
+						if ( self::cidr_match( $_SERVER['REMOTE_ADDR'], $range ) ) {
```

Comment:
Please either use `$this->` instead of `self::` with `protected function cidr_match( $ip, $range )` or change `cidr_match` to `static`